apps/src/studio/studio.js

```diff
@@ -1269,6 +1280,9 @@ studioApp.reset = function(first) {
     finishClipRect.setAttribute('x', Studio.spriteGoals_[i].x);
     finishClipRect.setAttribute('y', Studio.spriteGoals_[i].y);
   }
+
+  // A little flag for script-based code to consume.
+  window.studioLevelRestarted = true;
```

Comment:
Could you explain what this is used for? In general, adding items to the global namespace is undesirable. Also, if we tried to use this from our mochaTests we might get into trouble (since the global namespace in node is global instead of window).